src/test/scala/chiselTests/experimental/DataMirrorSpec.scala

```diff
@@ -55,4 +55,21 @@ class DataMirrorSpec extends ChiselFlatSpec {
     }
     ChiselStage.elaborate(new MyModule)
   }
+
+  it should "support getParent for normal modules" in {
+    class GrandChild(parent: RawModule) extends Module {
+      DataMirror.getParent(this) should be(Some(parent))
```

Comment:
oh nice, you can call this even before elaboration completes? that was more than i was hoping for...